docs/dunst.pod

```diff
@@ -142,6 +142,32 @@ the notification on the left border of the screen while a horizontal offset of
 
 =back
 
+=item B<no_progress_bar> (values: [true/false], default: false)
```

Comment:
Oh, I'd also like to avoid having negative no_* settings, I'd prefer if this is progress_bar=[true/false], defaulting to true.

Reply:
Yeah I like that better as well. The reason I did this was because it's not possible to change default true variables in command line. I don't think that will be used much in this case so I'll change it.